lsdcGui.py

```diff
@@ -2400,8 +2400,8 @@ def createSampleTab(self):
         self.sceneHutchTop.addItem(self.pixmap_item_HutchTop)
 
         self.pixmap_item.mousePressEvent = self.pixelSelect
-        if(daq_utils.defaultOverlayColor == 'GREEN'):
-          centerMarkBrush = QtGui.QBrush(QtCore.Qt.green)
+        if QtGui.QColor.fromString(daq_utils.defaultOverlayColor).isValid():
```

Comment:
shouldn't this still be restricted by the colors allowed by the settings? and I mean all of those colors should be available with the centerMarkBrush as well.

also think about how the value in the BlConfig parameter can be restricted as well.

Reply:
I have not chosen to restrict the color settings available if they are being set through the beamline configs, there will be a desynchronization between the drop-down menu color shown and the actual color displayed, but I assume anyone setting their own color to something different can realize it would cause that to happen.